aries-core/src/js/components/core/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Box } from 'grommet';
+
+export const Sidebar = ({ children, header, ...rest }) => {
+  return (
+    <Box
+      pad="medium"
+      gap="large"
+      fill="vertical"
+      width="auto"
+      height="100%"
```

Comment:
You don't need this with `fill="vertical"`

Reply:
right, this one is a left over...